spanner/spansql/keywords.go

```diff
@@ -158,6 +158,7 @@ var allFuncs = []string{
 	"BYTE_LENGTH", "CHAR_LENGTH", "CHARACTER_LENGTH",
 	"CODE_POINTS_TO_BYTES", "CODE_POINTS_TO_STRING",
 	"CONCAT",
+	"DATE",
```

Comment:
nit: Do not add this to the list of string functions (see comment above the list)

Reply:
Thanks!
Fixed and added more date timestamp functions at 0cb3b33e20d8b6add71d7f8e52c5d7fd60926a7e